### DISCUSSION TITLE
Rdkit doesn't retain DoublePosition parameter from cdxml molecular files

### DISCUSSION THREAD

**Discussion by Alesapo-py** [rdkit/rdkit#7998] [Q&A] [answered]
The "doubleposition" parameter typically specifies the spatial orientation of a double bond within a molecule. I need to preserve this information in an RDKit Mol object to ensure accurate molecular drawing. In my specific case, I'm working with a benzoimidazole molecule. The fused ring bond in this molecule is aromatic and should be oriented to the right, not the left. However, while the CDXML file indicates a "left" doubleposition, RDKit renders the double bond differently. I'm wondering if there's a way to make RDKit read and interpret the doubleposition parameter correctly. 
Thanks a lot for helping!

**Comment by bp-kelley** [comment, marked as answer]
We can certainly save it as a property on the bond, but without documentation it is quite hard to know what to do with this.

**Comment by bp-kelley** [comment]
Ok, this is what it does

<img width="244" alt="image" src="https://github.com/user-attachments/assets/b77ee026-548a-4c38-9cf4-3f8e5da0f288">

I'll admit that the RDKit doesn't care about these drawing preferences and will do nothing with them, but we could make an option to save certain CDX properties on bonds/atoms.   This might be useful if we ever write CDX.  However, if your goal is to duplicate ChemDraw rendering, the RDKit is probably not what you are looking for.

**Reply by bp-kelley** [reply]
However, if you have a complaint that the RDKit's rendering is double bonds in rings is sometimes not correct/pleasing to the eye, that might be another discussion/request.
